### PR TITLE
[DO NOT MERGE] short diff

### PR DESCRIPTION
**Pull request by Sabrina0614**

### CODE REVIEW

File: pkg/cluster/diff/diff.go [hunk 1]
```diff
@@ -233,6 +240,53 @@ func getFileLines(path string) ([]string, string, error) {
...
+			if strings.HasPrefix(trimmedLine, "labels") {
+				insideLabels = true
+				keep = true
+			} else if insideLabels {
+				log.Debug("leading spaces: ", strconv.Itoa(countLeadingSpaces(line)))
+				if countLeadingSpaces(line) == 2 || countLeadingSpaces(line) == 6 {
+					log.Debug("outside labels: ", line)
+					insideLabels = false
+				} else {
+					if strings.HasPrefix(trimmedLine, "k2.segment.com") || strings.HasPrefix(trimmedLine, "app.kubernetes.io/instance:") || strings.HasPrefix(trimmedLine, "version: ") {
```
Comment (yolken-segment) [inline pkg/cluster/diff/diff.go:277]: Fix line length here.

Comment (yolken-segment) [reply pkg/cluster/diff/diff.go:277]: Style nit- be consistent in the prefixes you're looking for, e.g. `k2.segment.com:`, `app.kubernetes.io/instance:`, `version:`, etc.

File: pkg/cluster/diff/diff.go [hunk 2]
```diff
@@ -233,6 +240,53 @@ func getFileLines(path string) ([]string, string, error) {
...
+			} else if insideAnnotation {
+				log.Debug("leading spaces: ", strconv.Itoa(countLeadingSpaces(line)))
+				// harcode labels here instead of checking the spaces
+				// since annotation is too long and it will then show
+				// some of the diff in annotation
+				if strings.HasPrefix(trimmedLine, "labels:") {
+					log.Debug("outside annotations: ", line)
+					insideAnnotation = false
+				} else {
+
```
Comment (yolken-segment) [inline pkg/cluster/diff/diff.go:260]: Remove excess vertical space.

File: pkg/cluster/diff/diff.go [hunk 3]
```diff
@@ -233,6 +240,53 @@ func getFileLines(path string) ([]string, string, error) {
...
 
+		if shortDiff && keep {
+			trimmedLine := strings.TrimLeft(line, " ")
+
+			// Skip over k2 annotations chunk in metadata since it's constantly
+			// changing and causing users to miss important changes.
+			if strings.HasPrefix(trimmedLine, "annotations:") {
+				insideAnnotation = true
+				keep = false
+			} else if insideAnnotation {
```
Comment (yolken-segment) [inline pkg/cluster/diff/diff.go:251]: Change to `insideAnnotations`.

File: pkg/cluster/diff/diff.go [hunk 4]
```diff
@@ -233,6 +240,53 @@ func getFileLines(path string) ([]string, string, error) {
...
+				log.Debug("leading spaces: ", strconv.Itoa(countLeadingSpaces(line)))
+				// harcode labels here instead of checking the spaces
```
Comment (yolken-segment) [inline pkg/cluster/diff/diff.go:253]: `harcode` -> `hardcode`

Comment (yolken-segment) [reply pkg/cluster/diff/diff.go:253]: I'm a little confused about why this is needed. Can you point me at an example?

File: pkg/cluster/diff/diff.go (hunk 1 above)
Comment (Sabrina0614) [reply pkg/cluster/diff/diff.go:277]: Will do! FYI, k2.segment.com/*, but will update the other two to be consistent.

Comment (yolken-segment) [reply pkg/cluster/diff/diff.go:277]: Ahh, ok, got it. Thanks!

**Comment by Sabrina0614** [comment]
@yolken-segment ready for another review when you get a chance. thanks

**Review by yolken-segment — COMMENTED** [review]
LGTM

**Pull request closed without merging**